Patch-Source: https://sources.debian.org/data/main/libt/libtest-unit-perl/0.25-7/debian/patches/40_make-deep-test-less-fragile.patch

Description: Add more variants to fragile test
 This test was already marked by upstream as "may be fragile due to
 recursion ordering" and it indeed is fragile. Fixes occasional test
 suite failures due to hash randomization.
Author: Axel Beckert <abe@debian.org>
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=87017

Index: libtest-unit-perl/t/tlib/AssertTest.pm
===================================================================
--- libtest-unit-perl.orig/t/tlib/AssertTest.pm	2013-12-26 02:08:38.782689704 +0100
+++ libtest-unit-perl/t/tlib/AssertTest.pm	2013-12-26 02:13:11.136010021 +0100
@@ -479,7 +479,8 @@
                  },
              }
          ],
-	 $differ->( 'HASH', 'not exist') => [$families{orig}, $families{bad_copy}], # test may be fragile due to recursion ordering?
+         # test is fragile due to hash randomization
+	 $differ->( 'HASH|John\sDoe', 'not\sexist|Baby\sDoll') => [$families{orig}, $families{bad_copy}],
 	 $differ->("'3'", "'5'") => [ [ \$H, 3 ], [ \$H2, 5 ] ],
 	 $differ->("'hello'", "'goodbye'") => [ { world => \$H }, { world => \$G } ],
 	 $differ->("'hello'", "'goodbye'") => [ [ \$H, "world" ], [ \$G, "world" ] ],
